src/Microsoft.ML.Data/Transforms/NormalizeColumn.cs

```diff
@@ -180,11 +180,12 @@ public abstract class ArgumentsBase : TransformInputBase
 
             public string TestType(ColumnType type)
             {
-                if (type.ItemType != NumberType.R4 && type.ItemType != NumberType.R8)
+                ColumnType itemType = type.GetItemType();
```

Comment:
>GetItemType [](start = 43, length = 11)

In most of the cases intent of this routine is to execute both if statements and return null. I would cast to vector here, feel free to switch statements.

Reply:
I switched them around and only cast to `VectorType` one time.

---
In reply to: [247766966](https://github.com/dotnet/machinelearning/pull/2131#discussion_r247766966) [](ancestors = 247766966)